### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jslavin-clearblade**
Clark,

For your review.

Thanks,

Jeff

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,26 +2,26 @@
...
 ## Applications
 
-CB Dev Kit currently supports transpilation of the following asset types:
+CB Dev Kit currently supports the transpilation of the following asset types:
 
-- Code Services - js and ts
-- Code Libraries - js and ts
-- Internal Resources (in portals) - js, ts, tsx, and jsx
-- Widget Parsers (in portals) - js, ts, tsx, and jsx
+- Code services: JS and TS
```
Comment (clarkbynum) [inline README.md:17]: it's ok to leave js, ts, jsx, and tsx lowercase. Those are file extensions

File: README.md [hunk 2]
```diff
@@ -63,33 +63,33 @@ cb-dev-kit init
...
 ##### Highlights
 
 - No need to look up flags and asset names/IDs
-- Selection of all asset types that can be created in src folder
+- Selection of all asset types that can be created in an src folder
 - Selection of all specific assets for each type
-- Selection of file types available for asset (js, ts, jsx, tsx)
-- Prompt for auto-generating adjacent unit test file for generated file
-- Prompt for auto-generating a React Component template for HTML widgets
+- Selection of file types available for the asset (JS, TS, JSX, TSX)
```
Comment (clarkbynum) [inline README.md:90]: Leave these lowercase

File: README.md [hunk 3]
```diff
@@ -108,28 +108,28 @@ cb-dev-kit create <options>
...
-| type             | (optional) file type of new source file. Options include js, jsx, ts, and tsx. Defaults to js | `cb-dev-kit create -portal=<portalname> -widgetId=<widgetId> -type=tsx`       |
+| widgetId         | Widget ID, source will include all available parsers                                          | `cb-dev-kit create -portal=<portalname> -widgetId=<widgetId>`                 |
+| internalResource | Internal resource name (including extension)                                                  | `cb-dev-kit create -portal=<portalname> -internalResource=<internalResource>` |
+| service          | Service name (without file extension)                                                         | `cb-dev-kit create -service=<serviceName> -type=ts`                           |
+| library          | Library name (without file extension)                                                         | `cb-dev-kit create -library=<libraryName> -type=ts`                           |
+| type             | (optional) file type of new source file. Options include JS, JSX, TS, and TSX. Defaults to JS | `cb-dev-kit create -portal=<portalname> -widgetId=<widgetId> -type=tsx`       |
```
Comment (clarkbynum) [inline README.md:115]: Leave file extensions lowercase

File: README.md [hunk 4]
```diff
@@ -169,18 +169,18 @@ cb-dev-kit clearblade-hot-reload start
...
 | noSSL       | Disables SSL for non-TLS connections on local systems. If on a production system with TLS enabled, ignore this flag and set -messagePort to 1884 | -noSSL=true              |
 | caPath      | If pointing at a production system and your certificate authority is not DigiCert, you must use -caPath to provide the absolute path of your CA. | -caPath=/\*/\*/\*/ca.pem |
 
-## Running Tests
+## Running tests
 
 ### Configuration
 
-- `cb-dev-kit init` will automatically install jest and provide the jest configuration in your system's package.json to check the src directory for test files, including babel configurations for jsx and tsx files
+- `cb-dev-kit init` will automatically install Jest and provide the Jest configuration in your system's package.json to check the src directory for test files, including Babel configurations for JSX and TSX files.
```
Comment (clarkbynum) [inline README.md:180]: Leave file extensions lowercase

**Review by clarkbynum — CHANGES_REQUESTED** [review]

**Review by jslavin-clearblade — COMMENTED** [review]
Clark,

I implemented your recommended changes.

Thanks,

Jeff

**Review by clarkbynum — APPROVED** [review]

**Pull request merged**